plugins/serializers/prometheus/convert.go

```diff
@@ -55,35 +55,30 @@ var LabelNameTable = Table{
 	},
 }
 
-func isValid(name string, table Table) bool {
-	if name == "" {
-		return false
```

Comment:
as we return true by default, did we lose the check for an empty sting with this? or was it checked elsewhere?

Reply:
yeah this would pass:
```
		{
			name: "empty name",
			metric: testutil.MustMetric(
				"",
				map[string]string{
					"host": "example.org",
				},
				map[string]interface{}{
					"time_idle": 42.0,
				},
				time.Unix(0, 0),
			),
			expected: []byte(`
_time_idle{host="example.org"} 42
`),

```

Reply:
I dug a bit and this would actually produce the same result on main since the only way to produce an empty metric name would be if the metric is called "prometheus". For empty tag this was a regression though, i added some tests and fixed your comments! thank you, i missed this